miner/README.org

```diff
@@ -207,12 +207,12 @@ it bad luck.
 
 *** I specify ~--chain=...~ but am getting work for other chains. Why?
 
-It is fundamental to the design of Chainweb that chains cannot progress much
-further than their neighbour chains. It may be that by asking for ~--chain=9~,
-the Node couldn't find work to do! In this case, it falls back to picking a
-random chain. This balances the needs of the Miner, who may want a specific
-Chain to progress efficiently, with the needs of the network, which requires all
-chains to grow evenly.
+It is fundamental to the design of the Kadena public blockchain that chains
+cannot progress much further than their neighbor chains. It may be that by
```

Comment:
"Neighbour" is International English spelling. Do we have company consensus on which to use?

Reply:
`neighbour` is definitely less common in the United States.

Reply:
Right - do we want US spellings or international ones?